Make sure the wp-secrets.php file is generated once and persisted

TrafeX opened the pull request:
I  moved the `wp-secrets.php` file to the volume mount next to wp-content to make this file persistent between container recreations. WIth a symlink I point WordPress to the right location.

Morpheus636 commented:
I like this solution, though with a couple things to note:

- The nginx config on this project *shouldn't* expose so-secrets.php in wp-content 
(it should attempt to run it instead and then 404, I believe) but I'm in favor of assuming that the wp-content directory is public just in case something got missed. 
- Would it be possible to retain the check for environment variables (the inner if statement) from #49? That will prevent warnings for deployments that used environment variables as a workaround for this bug.

Morpheus636 commented:
Update: I thought you were storing it *in* wp-content. Per the README, the volume is directly mounted to `/var/www/wp-content`, so files in /var/www/ shouldn't be persisted.

Morpheus636 commented:
Update: I thought you were storing it *in* wp-content. Per the README, the volume is directly mounted to `/var/www/wp-content`, so files in /var/www/ shouldn't be persisted.

TrafeX commented:
@Morpheus636 Thank you for the feedback!
You're right, it wasn't on the right location. I've moved it to wp-content. It's still safe because the Nginx config doesn't allow for files other than `index.php` to be executed. 

I also added the way you wrote the wp-secrets.php file so that the file is always created when it doesn't exist and not when the environment variables are set.

Morpheus636 reviewed (approved):
With the changes, LGTM!
